pkg/minikube/service/service.go

```diff
@@ -72,11 +71,11 @@ func (k *K8sClientGetter) GetCoreClient() (typed_core.CoreV1Interface, error) {
 // GetClientset returns a clientset
 func (*K8sClientGetter) GetClientset(timeout time.Duration) (*kubernetes.Clientset, error) {
 	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
-	profile := viper.GetString(config.MachineProfile)
+	contextName := config.GetMachineName()
```

Comment:
I'm not sure that the `context` prefix adds meaningful context to the variable name. What do you think about just `name` as a variable name?

https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Yes, Use `name` as a variable name is cleanly. Because of this variable will pass to the Context structure, so we don't add any prefix to emphasize this variable is a `context`.